tests/PasswordStrengthTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace CWP\Core\Tests;
+
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Security\Member;
+
+class PasswordStrengthTest extends SapphireTest
```

Comment:
I don't really agree with adding a test for this. I don't think it provides any benefit to test config settings.

Reply:
Indeed it appears to only be testing config settings, however that isn't the main goal of this minor test suite. The goal is more to catch 'regressions' should someone alter the values, given that the minimums tested here are a requirement for compliance.